test/include/utils.hpp

```diff
@@ -22,18 +20,35 @@
         EXPECT_EQ(actual, expected);                                    \
         const char* error_string;                                       \
         rgl_get_last_error_string(&error_string);                       \
-        EXPECT_THAT(error_string, HasSubstr(error_prefix));       \
-        EXPECT_THAT(error_string, HasSubstr(error_detail));         \
+        EXPECT_THAT(error_string, HasSubstr(error_prefix));       		\
+        EXPECT_THAT(error_string, HasSubstr(error_detail));         	\
```

Comment:
The purposes of `error_prefix` and `error_detail` seem the same, yet we have two different names.
An easy fix would be to rename both to `error_substring1`, `error_substring2`.
A more ambitious way would be to replace both with `__VA_ARGS__`.
Hint: something like this might work:
```cpp
EXPECT_RGL_STATUS(actual, expected, ...)
```
and then:
```cpp
for (auto&& substr : { __VA_ARGS__ }) { EXPECT_THAT(error_string, HasSubstr(substr)); }
```